packages/ds/src/components/select/select.stories.mdx

```diff
@@ -0,0 +1,158 @@
+import { Meta, Story, Props, Preview } from '@storybook/addon-docs/blocks';
+import { Box, Icons } from '../';
+import { Select } from './select';
+
+<Meta 
+  title="Components/Select"
+  component={Select}
+  parameters={{
+    a11y: {
+      config: {
+        rules: [{ id: 'label', enabled: false }],
+      },
+    },
+  }}
+/>
+
+# Select
+
+Form select component.
+
+When using this component, the menu options that you want `Select` component to display should be nested in using `option` tag. Specify `defaultValue` prop if you want `Select` to display an option other than the first one. If not specified, `Select` will display the first option by default.
+
+#### Default
+
+<Preview withToolbar>
+  <Story name="default">
+    <Box>
+      <Select
+        defaultValue="Select a city"
+        name="city"
+        mb={4}
+      >
+        <option>Select a city</option>
+        <option>Toronto</option>
+        <option>Istanbul</option>
+        <option>Amsterdam</option>
+      </Select>
+      <Select
+        defaultValue="Select a city"
+        leftIcon={<Icons.Place color="text.primary" aria-hidden />}
```

Comment:
The icon color is not working as expected. In has the wrong color in dark mode.

Reply:
Thanks for the catch, looking into it 👍